src/App.js

```diff
@@ -1,4 +1,4 @@
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] This function  should be class-based. The same way you did with the Calculator component. Please fix it.